src/pages/home/report/ReportActionItemMessageEdit.js

```diff
@@ -131,14 +137,23 @@ function ReportActionItemMessageEdit(props) {
         return () => {
             // Skip if this is not the focused message so the other edit composer stays focused.
             // In small screen devices, when EmojiPicker is shown, the current edit message will lose focus, we need to check this case as well.
-            if (!isFocusedRef.current && !EmojiPickerAction.isActive(props.action.reportActionID)) {
+            // When delete modal is opened, the current edit message will lose focus, we need to check this case as well
```

Comment:
The comments seems outdated, please update accordingly

Reply:
Updated